spec/swagger_helper.rb

```diff
@@ -18,6 +18,44 @@
       info: {
         title: "Teaching Vacancies ATS API",
         version: "v1",
+        description: <<~DESCRIPTION,
+          This document outlines the API’s key features, the data required for each listing,
+          and the steps needed to set up an integration.
+
+          This documentation describes each endpoint’s request parameters, response formats, and possible errors,
+          so you can seamlessly integrate Teaching Vacancies into your ATS workflow.
+
+          ## Introduction
+
+          The **Teaching Vacancies ATS API** enables you to manage job listings on behalf of schools or trusts
+          through your own Applicant Tracking System (ATS) or HR software.
+
+          By calling this API, you can:
+
+          - **List** all your active vacancies (with pagination)
+          - **Retrieve** details for a single vacancy
+          - **Create** new vacancies
+          - **Update** existing vacancies
+          - **Delete** vacancies when they’re no longer needed
```

Comment:
Suggestion up for discussion: I wonder if the `"they're no longer needed"` wording will make clients believe that they need to remove each vacancy at the end of their life.

Maybe something more explicit? EG:
```suggestion
          - **Delete** vacancies if they need to be removed before their expiration date
```